lib/compiler-finder.ts

```diff
@@ -203,6 +203,10 @@ export class CompilerFinder {
 
         const props: PropertyGetter = (propName: string, defaultValue?: any) => {
             const propsForCompiler = parentProps(langId, base + propName);
+            if (propsForCompiler && propName === 'options') {
```

Comment:
This seems like a very arbitrary and hardcoded, but also very big change to how properties work. I'm not in favour of this kind of thing: there shouldn't be special cases in the generic property system for a particularly-named option.

Reply:
Allow cascading properties for all `propName` was breaking the application, so the special case for `options` was added. But I have pushed a new commit to remove the special case and fix the problem in another way.